pkgs/top-level/all-packages.nix

```diff
@@ -35509,6 +35509,8 @@ with pkgs;
 
   soundtracker = callPackage ../applications/audio/soundtracker { };
 
+  spice-autorandr = callPackage ../applications/misc/spice-autorandr { };
```

Comment:
Can you use the new by-name structure https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#readme ?